spec/controllers/items_controller_spec.rb

```diff
@@ -71,8 +71,10 @@
   end
   describe 'embargo_update' do
     it 'should 403 if you are not an admin' do
-      allow(@current_user).to receive(:is_admin?).and_return(false)
-      post 'embargo_update', :id => @pid, :date => '12/19/2013'
+      expect(controller).not_to receive(:save_and_reindex)
+      expect(controller).not_to receive(:flush_index)
+      expect(controller).to receive(:forbid_modify).and_return(false)
```

Comment:
Should we submit the date format as it is created in the form?

<img width="323" alt="screen shot 2016-03-01 at 6 38 35 pm" src="https://cloud.githubusercontent.com/assets/1656824/13445873/e89cef0a-dfdc-11e5-8968-3b313e570ba2.png">